File: coding-practice/50 Coding Challenges part 1/Python/problem14.py
# find the first 10 Fibonacci numbers without using recursion
f0 = 0
f1 = 1

print("f0 = {}".format(f0))
print("f1 = {}".format(f1))

for i in range(2, 10):
    fi = f0 + f1
    print('f{} = {}'.format(i, fi))
    f0 = f1
    f1 = fi

File: coding-practice/50 Coding Challenges part 1/C#/problem14.cs
// coding challenge 14 : print the first 10 Fibonacci numbers without using recursion
using System;
class problem14
{
    static void print_fibonacci(int n_fibonacci){
        int f0 = 0;
        Console.Write(f0+" ");
        int f1 = 1;
        Console.Write(f1+" ");
        
        for(int i=2; i<n_fibonacci; i++){
            int fi = f1 + f0;
            Console.Write(fi+" ");
            f0 = f1;
            f1 = fi;
        }
    }

    static void Main(string[] args){
        int n_fibonacci = 10;
        print_fibonacci(n_fibonacci);
    }
}